vcpkg/VCPKG_COMMIT_SHA

```diff
@@ -1 +1 @@
-f3db66b403840b24ea2612d09cca30a5285f5ea3
+9ab3bafca4de5dec5a5a8b60f1d654cae7ad95aa
```

Comment:
The only drawback in the new version of VCPKG is that it contains new version of pglibrary, which needs newer `cmake` than it is installable normally with `sudo apt-get install cmake`. ubuntu server 20.10 has version: 3.16.3, but pglibrary wants:  3.17. Solution is to install `cmake` with `sudo snap install cmake --classic`.

I think I'll ignore that fact in my documentation, soon version ubuntu 21.04:).